Packages/com.unity.inputsystem/InputSystem/Controls/ButtonControl.cs

```diff
@@ -118,9 +132,50 @@ public new bool IsValueConsideredPressed(float value)
         /// </code>
         /// </example>
         /// </remarks>
-        public bool wasPressedThisFrame => device.wasUpdatedThisFrame && IsValueConsideredPressed(value) && !IsValueConsideredPressed(ReadValueFromPreviousFrame());
+        #if UNITY_EDITOR
+        public bool wasPressedThisFrame => InputUpdate.s_UpdateStepCount ==
```

Comment:
I find it odd that we have a separate editor state - but that's outside the scope of this change.

Reply:
Agreed, there is actually a lot of work done to separate interaction with game view vs editor while in play mode in general in many places.